tests/test_plugin_migration.py

```
@@ -0,0 +1,113 @@
+            
+            # Check MD file exists
+            md_path = plugin_dir / f"{category}.md"
+            self.assertTrue(md_path.exists(), f"MD file for {category} not found at {md_path}")
+            
+            # Check TOML file is valid by trying to load it
+            try:
+                loader.load_plugin(str(toml_path))
+                # If we get here, the plugin loaded successfully
+                plugin_loaded = True
+            except Exception as e:
+                plugin_loaded = False
```

Ruff (0.8.2)

Local variable `plugin_loaded` is assigned to but never used

Remove assignment to unused variable `plugin_loaded`

(F841)